shelley/chain-and-ledger/executable-spec/src/LedgerState.hs

```diff
@@ -435,26 +435,15 @@ validInputs tx u =
     else Invalid [BadInputs]
 
 -- |Implementation of abstract transaction size
-txsize
-  :: DSIGNAlgorithm dsignAlgo
-  => TxBody hashAlgo dsignAlgo
-  -> Integer
+txsize :: TxBody hashAlgo dsignAlgo -> Integer
 txsize = toEnum . length . show
 
 -- |Minimum fee calculation
-minfee
-  :: DSIGNAlgorithm dsignAlgo
-  => PParams
-  -> TxBody hashAlgo dsignAlgo
-  -> Coin
+minfee :: PParams -> TxBody hashAlgo dsignAlgo -> Coin
 minfee pc tx = Coin $ pc ^. minfeeA * txsize tx + fromIntegral (pc ^. minfeeB)
```

Comment:
Unrelated to the current PR, but should we add the size of the witness maps here, too? Else redundant signatures will not increase the minimally required fee, which is probably not what we want